get-started/05_persisting_data.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "Persisting our DB"
+keywords: get started, setup, orientation, quickstart, intro, concepts, containers, docker desktop
+description: Making our DB persistent in our application
+---
+
+In case you didn't notice, our todo list is being wiped clean every single time
+we launch the container. Why is this? Let's dive into how the container is working.
+
+## The Container's Filesystem
+
+When a container runs, it uses the various layers from an image for its filesystem.
+Each container also gets its own "scratch space" to create/update/remove files. Any
+changes won't be seen in another container, _even if_ they are using the same image.
+
+### Seeing this in Practice
+
+To see this in action, we're going to start two containers and create a file in each.
+What you'll see is that the files created in one container aren't available in another.
+
+1. Start a `ubuntu` container that will create a file named `/data.txt` with a random number
+   between 1 and 10000.
+
+    ```bash
+    docker run -d ubuntu bash -c "shuf -i 1-10000 -n 1 -o /data.txt && tail -f /dev/null"
+    ```
+
+    In case you're curious about the command, we're starting a bash shell and invoking two
+    commands (why we have the `&&`). The first portion picks a single random number and writes
+    it to `/data.txt`. The second command is simply watching a file to keep the container running.
+
+1. Validate we can see the output by `exec`'ing into the container. To do so, open the Dashboard and click the first action of the container that is running the `ubuntu` image.
+
+    ![Dashboard open CLI into ubuntu container](images/dashboard-open-cli-ubuntu.png){: style=width:75% }
+{: .text-center }
+
+    You will see a terminal that is running a shell in the ubuntu container. Run the following command to see the content of the `/data.txt` file. Close this terminal afterwards again.
+
+    ```bash
+    cat /data.txt
+    ```
+
+    If you prefer the command line you can use the `docker exec` command to do the same. You need to get the
+   container's ID (use `docker ps` to get it) and get the content with the following command.
+
+    ```bash
+    docker exec <container-id> cat /data.txt
+    ```
+
+    You should see a random number!
+
+1. Now, let's start another `ubuntu` container (the same image) and we'll see we don't have the same
```

Comment:
The numbered list is slightly off here. Perhaps try pasting as plain text and then format?

Reply:
Interesting in VS preview it is working fine for me